### PR TITLE
call close issue resolve

### PR DESCRIPTION
**Pull request by helenKaryamsetty**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.  

# Type of change  

- [x] Bug fix  
- [ ] New feature  
- [ ] Enhancement  
- [ ] Refactoring  
- [ ] Documentation  
- [ ] Other ( please specify )  

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/main/java/com/iemr/common/config/InterceptorConfig.java [hunk 1]
```diff
@@ -35,15 +35,15 @@ public class InterceptorConfig extends WebMvcConfigurerAdapter
 	@Autowired
 	HTTPRequestInterceptor requestInterceptor;
 
-//	@Override
-//	public void addInterceptors(InterceptorRegistry registry)
-//	{
-//		registry.addInterceptor(requestInterceptor);
-//	}
+	// @Override
```
Comment (drtechie) [inline src/main/java/com/iemr/common/config/InterceptorConfig.java:38]: Remove commented-out code.

File: src/main/java/com/iemr/common/service/cti/CTIServiceImpl.java [hunk 2]
```diff
@@ -573,7 +573,9 @@ public OutputResponse getVoiceFile(String request, String remoteAddr) throws IEM
 			disposition.setResponse(ctiResponse);
 			output.setResponse(disposition.toString());
 		} else {
-			output.setError(OutputResponse.GENERIC_FAILURE, ctiResponse.getReason(), ctiResponse.getStatus());
+			output.setResponse("path not found");
+			logger.info("URL " + ctiURI + " returned response " + response + " and saved response - path not found");;
+//			output.setError(OutputResponse.GENERIC_FAILURE, ctiResponse.getReason(), ctiResponse.getStatus());
```
Comment (drtechie) [inline src/main/java/com/iemr/common/service/cti/CTIServiceImpl.java:578]: Remove commented-out code.

**Review by drtechie — APPROVED** [review]

**Comment by drtechie** [comment]
@helenKaryamsetty please add the properties to CI file too.

File: src/main/environment/common_dev.properties [hunk 3]
```diff
@@ -55,6 +55,11 @@ cron-scheduler-email=0 0/1 * * * ? *
 start-ctidatasync-scheduler=false
 cron-scheduler-ctidatasync=0 00 01 * * ? *
 
+###cti data check with call detail report Scheduler
+#Runs at everyday 12:10AM 
+start-ctidatacheck-scheduler=true
```
Comment (drtechie) [inline src/main/environment/common_dev.properties:60]: Add these to common_ci.properties too.

**Review by ravishanigarapu — APPROVED** [review]
looks fine

**Pull request merged**